pexpect/pxssh.py

```diff
@@ -294,8 +294,15 @@ def login (self, server, username, password='', terminal_type='ansi',
         session to do so. Setting this option to `False` and not having an active session
         will trigger an error.
         
-        Set ``ssh_key`` to `True` to force passing the current SSH authentication socket to the
+        Set ``ssh_key`` to a file path to an SSH private key will use that SSH key
+        for the session authentication.
+        Set ``ssh_key`` to `True` to force passing the current SSH authentication socket
         to the desired ``hostname``.
+        
+        Set ``ssh_config`` to a file path string of an SSH client config file will pass that
```

Comment:
These need another minor tweak to fix the grammar for "Set" rather than "Setting":

- Setting X to Y **will** do Z
- Set X to Y **to** do Z

Reply:
Fair enough.